### PR TITLE
Remove python2 from buildroot

### PR DESCRIPTION
**Pull request by BryanQuigley**
I was able to build images fine without it and don't see any scripts that need it.

### CODE REVIEW

**Comment by dbnicholson** [comment]
I had a suspicion that python2 was still used in some esoteric setup, and it is in our split image tool. We're actually phasing it out, but in the meantime #22 fixes it. After that's merged I think we can merge this.

**Review by jprvita — APPROVED** [review]
LGTM, and after #22 I can't find any other use of python2 in this tool (and if some corner case breaks, we should fix it to use python3 anyway).

**Comment by jprvita** [comment]
Thanks for the contribution @BryanQuigley!

@dbnicholson Feel free to merge this PR when you have a chance.

**Pull request merged**
